### PR TITLE
clone keycloak chart

### PR DESCRIPTION
**Pull request by robert-nemet**
Adding keycloak from codecentric, because we are already using it. When using it with ArgoCD image updater and digest strategy the label ` app.kubernetes.io/version` can have a value like this `"master@sha256:72ef973fd6d3f3e6cea8a9e1b5cbac6daf1bf8cc3d8871cbd"` which is not valid. That is why we are fixing definition for `keycloak.labels` in the `charts/keycloak/templates/_helpers.tpl`. Everything is the same as in codecentric chart.

Link to the change: https://github.com/ricardo-ch/helm-charts/blob/006151c6bdd7eaea80cfed9639597667c5a45006/charts/keycloak/templates/_helpers.tpl#L40

### CODE REVIEW

**Comment by julien-may** [comment]
as this is mainly a copy of the files from codecentric, may i ask you point to the location of where you made the actual changes? this would greatly help to do an actual review

**Comment by robert-nemet** [comment]
> as this is mainly a copy of the files from codecentric, may i ask you point to the location of where you made the actual changes? this would greatly help to do an actual review

Updated description.

File: charts/keycloak/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "keycloak.labels" -}}
+helm.sh/chart: {{ include "keycloak.chart" . }}
+{{ include "keycloak.selectorLabels" . }}
+app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | replace ":" "_" | replace "@" "_" | trunc 63 | quote }}
```
Comment (aaronstutzer) [inline charts/keycloak/templates/_helpers.tpl:40]: 👍

**Review by bueti — APPROVED** [review]
lgtm.

**Comment by bueti** [comment]
are you going to open a PR at the upstream chart so we can get rid of this one sometime in the future?

**Pull request merged**